internal/support/logging/file.go

```diff
@@ -99,7 +120,11 @@ func (fl *fileLog) remove(criteria matchCriteria) (int, error) {
 	return count, nil
 }
 
-func (fl *fileLog) find(criteria matchCriteria) ([]models.LogEntry, error) {
+func (fl *fileLog) FindLog(criteria db.LogMatcher, limit int) ([]models.LogEntry, error) {
```

Comment:
A testPersistenceFind should be added. The limit is not working in the files backend. It is the cause of the other test returning StatusRequestEntityTooLarge.